### DISCUSSION TITLE
Edit Only, prevent adding new content

### DISCUSSION THREAD

**Discussion by brendenrdowd** [react-page/react-page#1027] [Q&A]
I'm looking for a editor that will allow the end user to rearrange/move existing elements in a component, without the ability to add new elements. I was looking at the custom plugins to see if it was possible to disable adding elements but I didn't see anything definitive, is this possible using react-page?

**Comment by macrozone** [comment]
we could add an option to only allow the layout mode.

You can imaging that by going to https://react-page.github.io/ and select "move blocks":

<img width="80" alt="Bildschirmfoto 2021-06-11 um 13 54 58" src="https://user-images.githubusercontent.com/1972353/121682631-9d7fe880-cabc-11eb-939d-ce10bc3310b7.png">


But i would like to hear to the usecase behind the requirement first to find the best solution.
